db/seeds.rb

```diff
@@ -5,3 +5,171 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+movies = Movie.create([
+  {
+    name: 'Encanto',
+    poster: 'https://lumiere-a.akamaihd.net/v1/images/encanto_p_243_ster_29_23e974a4.jpeg',
+    start_date: '2021-11-27',
```

Comment:
Aquí igualmente, sería bueno usar `Date.today` y luego `Date.today.advance(days: X)` para en end date,
Créditos a @dhvasquez por usarlo en su `FactoryBot`

Reply:
thanks for the comment